dev-packages/cli/src/theia.ts

```diff
@@ -156,6 +156,12 @@ function rebuildCommand(command: string, target: ApplicationProps.Target): yargs
                     describe: 'Controls whether to pack or unpack plugins',
                     boolean: true,
                     default: false,
+                },
+                'tolerate_download_failures': {
+                    alias: 'tolerate-download-failures',
```

Comment:
`alias` should be shorter (`failOnError`):

```suggestion
                    alias: 'f',
```